test.js

```diff
@@ -117,6 +118,75 @@ test('camelCase with pascalCase option', t => {
 	t.is(camelCase('1hello', {pascalCase: true}), '1Hello');
 	t.is(camelCase('1Hello', {pascalCase: true}), '1Hello');
 	t.is(camelCase('h1W', {pascalCase: true}), 'H1W');
+	t.is(camelCase('label-name-SC', {pascalCase: true}), 'LabelNameSc');
+});
+
+test('camelCase with saveCapital option', t => {
+	t.is(camelCase('foo', {saveCapital: true}), 'foo');
+	t.is(camelCase('foo-bar', {saveCapital: true}), 'fooBar');
+	t.is(camelCase('foo-bar-baz', {saveCapital: true}), 'fooBarBaz');
+	t.is(camelCase('foo--bar', {saveCapital: true}), 'fooBar');
+	t.is(camelCase('--foo-bar', {saveCapital: true}), 'fooBar');
+	t.is(camelCase('--foo--bar', {saveCapital: true}), 'fooBar');
+	t.is(camelCase('FOO-BAR', {saveCapital: true}), 'FOOBAR');
```

Comment:
Can you add tests cases for `foo-baR`, `foo-BaR`, `foo-Bar`, `foo-BAr`, `foo-B`, `foo-2-BAR`, `foo-2BAR`, `foo-BAR2`, `foo-BAR2BAR`?